src/sider/historikk/container/HistorikkContainer.tsx

```diff
@@ -184,6 +190,45 @@ function createHistorikkEventsFromFriskTilArbeid(
   });
 }
 
+function dialogmoteText(dialogmote: DialogmoteDTO): string {
+  switch (dialogmote.status) {
+    case DialogmoteStatus.INNKALT:
+      return `Det ble innkalt til dialogmøte ${tilDatoMedManedNavnOgKlokkeslett(
+        dialogmote.tid
+      )} av ${dialogmote.opprettetAv}`;
+    case DialogmoteStatus.AVLYST:
+      return `Dialogmøtet den ${tilDatoMedManedNavnOgKlokkeslett(
+        dialogmote.tid
+      )} ble avlyst av ${dialogmote.opprettetAv}`;
+    case DialogmoteStatus.FERDIGSTILT:
+      return `Det ble skrevet referat for dialogmøtet avholdt ${tilDatoMedManedNavnOgKlokkeslett(
+        dialogmote.tid
+      )} av ${dialogmote.opprettetAv}`;
+    case DialogmoteStatus.NYTT_TID_STED:
+      return `Dialogmøtet ble endret til nytt tid eller sted av ${dialogmote.opprettetAv}`;
+    case DialogmoteStatus.LUKKET:
+      return `Dialogmøtet den ${tilDatoMedManedNavnOgKlokkeslett(
+        dialogmote.tid
+      )} ble manuelt lukket av systemet`;
+  }
+}
+
+function createHistorikkEventsFromDialogmote(
+  dialogmoter: DialogmoteDTO[]
+): HistorikkEvent[] {
+  return dialogmoter.map((dialogmote: DialogmoteDTO) => {
+    return {
+      opprettetAv: dialogmote.opprettetAv,
+      tekst: dialogmoteText(dialogmote),
+      tidspunkt:
```

Comment:
Vil ikke `dialogmoter` her bare inneholde alle dialogmøter som er innkalt for personen med siste status? Så for et ferdigstilt møte vil man f.eks. bare få ett innslag i historikk med når det ble ferdigstilt :thinking: